### PR TITLE
fribidi: fix cross compilation

### PR DESCRIPTION
**Pull request by tadeokondrak**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by tadeokondrak** [comment]
Hard to switch the target branch of a PR without causing review noise, sorry :/

File: pkgs/development/libraries/fribidi/default.nix [hunk 1]
```diff
@@ -33,13 +31,14 @@ stdenv.mkDerivation rec {
...
+  nativeBuildInputs = [ meson ninja pkgconfig ]
+    ++ stdenv.lib.optional stdenv.isDarwin fixDarwinDylibNames;
 
-  checkInptus = [ python3 ];
+  doCheck = true;
+  checkInputs = [ python3 ];
 
   meta = with stdenv.lib; {
-    homepage = https://github.com/fribidi/fribidi;
+    homepage = "https://github.com/fribidi/fribidi";
```
Comment (FRidh) [inline pkgs/development/libraries/fribidi/default.nix]: Unnecessary change.

**Review by FRidh — CHANGES_REQUESTED** [review]
